packages/ember-application/tests/system/dependency_injection/default_resolver_test.js

```diff
@@ -64,9 +65,15 @@ QUnit.test('the default resolver looks up templates in Ember.TEMPLATES', functio
   setTemplate('fooBar', fooBarTemplate);
   setTemplate('fooBar/baz', fooBarBazTemplate);
 
-  equal(locator.lookupFactory('template:foo'), fooTemplate, 'resolves template:foo');
-  equal(locator.lookupFactory('template:fooBar'), fooBarTemplate, 'resolves template:foo_bar');
-  equal(locator.lookupFactory('template:fooBar.baz'), fooBarBazTemplate, 'resolves template:foo_bar.baz');
+  if (isFeatureEnabled('ember-factory-for')) {
+    equal(locator.factoryFor('template:foo').class, fooTemplate, 'resolves template:foo');
+    equal(locator.factoryFor('template:fooBar').class, fooBarTemplate, 'resolves template:foo_bar');
+    equal(locator.factoryFor('template:fooBar.baz').class, fooBarBazTemplate, 'resolves template:foo_bar.baz');
+  } else {
+    equal(locator.lookupFactory('template:foo'), fooTemplate, 'resolves template:foo');
```

Comment:
It is concerning that this test coverage is lost when the feature flag is enabled?